### PR TITLE
Redesign the periodic inbox health email

### PR DESCRIPTION
**Pull request by elie222**
Stacked on #3134. Review that one first; this PR's diff is only the email.

## Changes

Rebuilds the rarely read senders email to the approved design.

- Card layout on a tinted page, brand header, single primary action
- Each sender row carries a bar sized by how much of that sender's mail goes unread, so the ranking is visible rather than implied
- Rows report an opened ratio ("opened 1 in 9", "opened none") instead of a read percentage. It reads faster and stays honest for senders never opened at all, where a percentage renders as a bare 0%
- The summary line reports the weekly volume the user never opens, computed from actual unread counts rather than derived from the yearly total, so the sentence is literally true
- List heading and the trailing "more senders" line adapt to how many senders are listed, so copy stays correct at the minimum threshold and when every suggestion already fits in the list
- Footer carries the postal address

The header uses the app icon plus the brand name, matching the other emails in this package, rather than the wordmark treatment in the design source.

## Data

`getInboxHealthEmailData` now returns per-sender read and ignored counts plus a weekly ignored figure. The API route spreads the helper's result into the email props, so no call-site changes were needed.

## Testing

- Rendered the component through `@react-email/render` and compared against the design at desktop and mobile widths, plus the edge case where every suggestion is listed and the trailing line is suppressed
- Verified the responsive stacking rules and MSO line-height hints survive the React render
- Helper tests updated for the new fields, with coverage for the weekly figure
- Lint clean

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: packages/resend/emails/inbox-health.tsx [hunk 1]
```diff
@@ -29,94 +14,397 @@ export interface InboxHealthEmailProps {
...
+                      border={0}
+                    >
+                      <tr>
+                        <td
+                          style={{
+                            paddingRight: "8px",
+                            verticalAlign: "middle",
+                          }}
+                        >
+                          <a href={baseUrl}>
```
[the remark this answers is not in the document]
Comment (elie222) [reply packages/resend/emails/inbox-health.tsx]: Valid, fixed in 1766fc195.

The icon and the brand name were both linked to the homepage, so the icon rendered as a second link to the same destination with no accessible name (`alt=""`). Removed the anchor from the icon and left it decorative, so the brand name is now the single labelled link and its accessible name is the visible text.

An anchor can't span two table cells, so wrapping both in one link wasn't an option in this layout. Verified the rendered output now contains exactly one homepage link.
